samples/wifi/provisioning/sample_description.rst

```diff
@@ -0,0 +1,93 @@
+.. _wifi_provisioning_sample_desc:
+
+Sample description
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Wi-Fi Provisioning Service sample demonstrates how to provision a device with Nordic Semiconductor's Wi-Fi chipsets over Bluetooth® Low Energy.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+The sample requires a smartphone (configurator) with Nordic Semiconductor's nRF Wi-Fi Provisioner app installed in one of the following versions:
+
+* `nRF Wi-Fi Provisioner mobile app for Android`_
+* `nRF Wi-Fi Provisioner mobile app for iOS`_
+
+Overview
+********
+
+With this sample, you can provision a Wi-Fi device that lacks input or output capability.
+The sample is divided into three parts:
+
+* Task and event handling component - It handles provisioning related tasks and events.
```

Comment:
@richabp, I'm not sure on the list formatting, but 'it handles' reads strangely to me. 

A little more natural is 'handles provisioning-related tasks and events.' without a pronoun. Same for other components.